src/plugins/event_annotation/server/plugin.ts

```diff
@@ -16,7 +16,6 @@ import {
   queryPointEventAnnotation,
 } from '../common';
 import { setupSavedObjects } from './saved_objects';
-// import { getFetchEventAnnotations } from './fetch_event_annotations';
```

Comment:
Is it needed for the follow up PR?

Reply:
I didn't add it. Didn't look like was in use, so thats why I removed it here.